### PR TITLE
added_separate_mem_flag

### PR DESCRIPTION
**Pull request by tczene**
IoT.js-DCO-1.0-Signed-off-by: Tamás Czene tczene.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by tczene** [comment]
Added new flag to build configuration to use ccm memory.
    More details: #306

**Comment by chunseoklee** [comment]
@tczene please, wait for merging https://github.com/Samsung/jerryscript/pull/732. Then, need to update jerry submodule.

**Comment by egavrin** [comment]
@chunseoklee I merged #732 to master.

**Comment by ILyoan** [comment]
@chunseoklee please review this one

**Comment by LaszloLango** [comment]
LGTM

File: tools/build.py [hunk 1]
```diff
@@ -189,6 +189,8 @@ def init_option():
 
     parser.add_argument('--jerry-lto', action='store_true')
 
+    parser.add_argument('--use-ccm', action='store_true')
```
Comment (zherczeg) [inline tools/build.py]: I don't like this ccm name. I would prefer a section
--jerry-heap-section

**Comment by LaszloLango** [comment]
The source repository of this PR does not exits, so cannot be updated.

**Pull request closed without merging**